lib/rubocop/cop/rails/active_model_errors_direct_manipulation.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks direct manipulation of ActiveModel#errors as hash.
```

Comment:
Is it `ActiveModel::Errors` or `ActiveModel::Validators#errors`?

```suggestion
      # This cop checks direct manipulation of `ActiveModel::Errors` as hash.
```

Reply:
The method is defined in `ActiveModel::Validations#errors`, but people mostly know it as `model.errors`. It is also a class of `ActiveModel::Errors`.  So both are correct.